example/demo_generator/src/server.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2025, Polytechnique Montreal and contributors
+ *
+ * This file is licensed under the MIT License.
+ * License text available at https://opensource.org/licenses/MIT
+ */
+import path from 'path';
+
+import setupServer from 'evolution-backend/lib/apps/participant/index';
+import { setProjectConfig } from 'evolution-backend/lib/config/projectConfig';
+import { registerTranslationDir, addTranslationNamespace } from 'chaire-lib-backend/lib/config/i18next';
+import roleDefinitions from './survey/server/roleDefinition';
+import customFilterForValidation from './survey/server/serverConfiguration';
+import serverUpdateCallbacks from './survey/server/serverFieldUpdate';
+import serverValidations from './survey/server/serverValidations';
+
+const configureServer = () => {
+    // Default values for each field. Same as default config, but this is an example project, keep it here.
+    setProjectConfig({
+        serverUpdateCallbacks,
+        serverValidations,
+        roleDefinitions,
+        validationListFilter: customFilterForValidation
```

Comment:
Ceci devrait aller seulement dans l'admin, non?<

D'ailleurs tous les project configs ici devraient aussi être dans l'admin, qui lui a été initialisé à rien de particulier (ce qui est aussi correct à ce stade-ci), mais il faudrait que les 2 soient identiques. Aussi, je ne crois pas que le commentaire `// Default values for each field` soit correct ici.